### ISSUE TITLE
Mr

### ISSUE THREAD

**Issue by geoxx3** [bbc/dialogger#2]
Hi,
I would like to use Dialogger for a voice to text transcription project.
I have problems installing. Can you help me with a newer version?
Thanks for understanding,

George Rusu

**Comment by chrisbaume** [comment]
Hi George. Can you please explain what you're trying to achieve? As you'll have seen in the README, this project does not include a transcription service - you have to integrate it with something else.

Have you considered commercial solutions such as [Trint](https://trint.com/) or [Descript](https://www.descript.com/)?

**Comment by geoxx3** [comment]
Hi Chris,

Thanks for your prompt response.

I did the transcription with the Google API. I would like the transcribed file to be edited to increase the accuracy of the transcript. For this I need an editor with audio / video player and text transcription as you have the logic diagram. I don't have the text editing part with the player.

I would like to do something like www.sonix.com <http://www.sonix.com>  or www.happyscribe.com <http://www.happyscribe.com>  which also have an editor for transcribed files.

I found your software on GitHub and I'm trying to use it in my project but I couldn't install it. Thanks for what you recommended to me. Looks like I could use the API from www.descript.com <http://www.descript.com> .

Thank you so much for your understanding,

 

Best regards,

George
